lib/date_range.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'hotel_controller'
```

Comment:
This creates what's called a circular load, think about your code like a tree. Some things are going to be closer to the root, some things are going to be the ends of branches. This file is the end of a branch!